modules/lvms-using-raid-arrays.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-using-raid-arrays_{context}"]
+= Using software RAID arrays in {lvms}
+
+// Logical Volume Manager (LVM) does not support creating a software RAID.
+
+// You can create the RAID arrays using the `mdadm` utility. You can also use the RAID arrays that are created using the `mdadm` utility in LVM Storage. 
+
+Logical Volume Manager (LVM) does not support creating a software RAID. However, you can create the RAID arrays using the `mdadm` utility, and use the RAID arrays in the LVM Storage.
+
+.Prerequisites
+
+* You have created a software RAID during the {product-title} installation.
++
+[NOTE]
+====
+You can create the RAID arrays only during the installation of {product-title}. For information about creating RAID arrays in {product-title}, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html/installing/installation-configuration#installation-special-config-raid_installing-customizing[Configuring a RAID-enabled data volume]
+====
+
+* You have installed LVM Storage.
+
+.Procedure
+
+. Open the `LVMCluster` custom resource (CR) YAML file.
++
+[NOTE]
+====
+If you have created the `LVMCluster`, then you can edit the existing `LVMCluster` CR. Else, you can perform this step while creating a new `LVMCluster`.
+====
+
+. Add the path to the RAID array in the `deviceSelector` section in `LVMCluster` custom resource (CR) YAML.
++
+.Example LVMCluster CR
+[source,yaml]
+----
+apiVersion: lvm.topolvm.io/v1alpha1
+kind: LVMCluster
+metadata:
+  name: my-lvmcluster
+spec:
+  storage:
+    deviceClasses:
+    - name: vg1
+      default: true
+      deviceSelector: <1>
```

Comment:
This should be an example of a path for a mdadm raid device

Reply:
I suggest removing the other paths for clarity

Reply:
Updated as per the comment.